go/noise/noise.go

```diff
@@ -57,6 +59,17 @@ func ToKind(n Noise) Kind {
 	return GaussianNoise
 }
 
+// ConfidenceInterval holds lower and upper bounds as float64 for the confidence interval.
+type ConfidenceInterval struct {
+	LowerBound, UpperBound float64
+}
+
+// toConfidenceIntervalInt64 converts a ConfidenceInterval struct to int64, by
+// rounding the lower and upper bounds.
```

Comment:
This doesn't convert to `int64` now, so maybe change this to something like
```
toConfidenceIntervalInt64 rounds the lower and upper bounds of a ConfidenceInterval struct for integer valued noise operations.
```

Extra points if you can come up with a better comment. :)